src/pages/NewChatPage.js

```diff
@@ -77,7 +77,16 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, translate, i
         const sectionsList = [];
         let indexOffset = 0;
 
-        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(searchTerm, selectedOptions, filteredRecentReports, filteredPersonalDetails, {}, false, indexOffset);
+        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(
+            searchTerm,
+            selectedOptions,
+            filteredRecentReports,
+            filteredPersonalDetails,
+            {},
+            false,
+            indexOffset,
+            maxParticipantsReached,
```

Comment:
For example, if we swap the param order, this could be shortened to:

```suggestion
            indexOffset,
            maxParticipantsReached,
```